MarsSpecFlowProject/MarsSpecFlowProject/StepDefinitions/ThisTestSuiteContainsTestScenariosForLanguageTab_StepDefinitions.cs

```diff
@@ -45,13 +48,176 @@ public void GivenILogIntoThePortalWithUserNameAndPassword(string UserName, strin
         }
 
 
-        
+
+
+        [Given(@"User has no language in their profile")]
+        public void GivenUserHasNoLanguageInTheirProfile()
+        {
+            langobj.DeleteAllElements(driver);
+            Thread.Sleep(3000);
+        }
+
+        [When(@"I create a new language record '([^']*)' '([^']*)'")]
+        public void WhenICreateANewLanguageRecord(string Language, string Level)
+        {
+            langobj.Addlanguage(driver, Language, Level);
```

Comment:
dont pass the driver around in action methods, instead create a common driver class which injects the driver and then extend the pages to that class!

Reply:
please change this everywhere

Reply:
make sure you apply this to all steps